### ISSUE TITLE
No license information for images

### ISSUE THREAD

**Issue by AntiSC2** [Kruptein/PlanarAlly#22]
Could not find any license information about any of the images included in this project.

**Comment by Kruptein** [comment]
Hey, my bad, those images are not supposed to be in the actual repository as I indeed do not own the permission to distribute those.

(You are talking about the two assets right?)

**Comment by AntiSC2** [comment]
Yes, but they are some other images like login_background.png and such in this project that I do think you own. So it would be great if you could at least mention them in the license file or something.
